fabtests/pytest/common.py

```diff
@@ -223,18 +291,18 @@ def __init__(self, cmdline_args, executable,
 
         self._cmdline_args = cmdline_args
         self._timeout = timeout or cmdline_args.timeout
-        self._server_base_command = self.prepare_base_command("server", executable, iteration_type,
+        self._server_base_command, server_additonal_environment = self.prepare_base_command("server", executable, iteration_type,
```

Comment:
What is the advantage of returning a separate argument instead of combining both into a single command?

Reply:
The ClientServerTest does not have enough information to prepare full command. For example, it does not have the path to the binary, which need to be added to the beginning of the base_command